app-nia-catalog/dependencies/releaseRuntimeClasspath.txt

```diff
@@ -1,56 +1,58 @@
-androidx.compose.material:material-ripple-android:1.5.4
-androidx.compose.material:material-ripple:1.5.4
-androidx.compose.runtime:runtime-android:1.5.4
-androidx.compose.runtime:runtime-saveable-android:1.5.4
-androidx.compose.runtime:runtime-saveable:1.5.4
-androidx.compose.runtime:runtime:1.5.4
-androidx.compose.ui:ui-android:1.5.4
-androidx.compose.ui:ui-geometry-android:1.5.4
-androidx.compose.ui:ui-geometry:1.5.4
-androidx.compose.ui:ui-graphics-android:1.5.4
-androidx.compose.ui:ui-graphics:1.5.4
-androidx.compose.ui:ui-text-android:1.5.4
-androidx.compose.ui:ui-text:1.5.4
-androidx.compose.ui:ui-tooling-preview-android:1.5.4
-androidx.compose.ui:ui-tooling-preview:1.5.4
-androidx.compose.ui:ui-unit-android:1.5.4
-androidx.compose.ui:ui-unit:1.5.4
-androidx.compose.ui:ui-util-android:1.5.4
-androidx.compose.ui:ui-util:1.5.4
-androidx.compose.ui:ui:1.5.4
-androidx.compose:compose-bom:2023.10.01
+androidx.collection:collection-jvm:1.4.0
+androidx.collection:collection-ktx:1.4.0
+androidx.collection:collection:1.4.0
+androidx.compose.animation:animation-android:1.6.1
+androidx.compose.animation:animation-core-android:1.6.1
+androidx.compose.animation:animation-core:1.6.1
+androidx.compose.animation:animation:1.6.1
+androidx.compose.foundation:foundation-android:1.6.1
+androidx.compose.foundation:foundation-layout-android:1.6.1
+androidx.compose.foundation:foundation-layout:1.6.1
+androidx.compose.foundation:foundation:1.6.1
+androidx.compose.material3:material3-android:1.2.0
+androidx.compose.material3:material3:1.2.0
+androidx.compose.material:material-icons-core-android:1.6.1
+androidx.compose.material:material-icons-core:1.6.1
+androidx.compose.material:material-icons-extended-android:1.6.1
+androidx.compose.material:material-icons-extended:1.6.1
+androidx.compose.material:material-ripple-android:1.6.1
+androidx.compose.material:material-ripple:1.6.1
+androidx.compose.runtime:runtime-android:1.6.1
+androidx.compose.runtime:runtime-saveable-android:1.6.1
+androidx.compose.runtime:runtime-saveable:1.6.1
+androidx.compose.runtime:runtime:1.6.1
+androidx.compose.ui:ui-android:1.6.1
+androidx.compose.ui:ui-geometry-android:1.6.1
+androidx.compose.ui:ui-geometry:1.6.1
+androidx.compose.ui:ui-graphics-android:1.6.1
+androidx.compose.ui:ui-graphics:1.6.1
+androidx.compose.ui:ui-text-android:1.6.1
+androidx.compose.ui:ui-text:1.6.1
+androidx.compose.ui:ui-tooling-preview-android:1.6.1
+androidx.compose.ui:ui-tooling-preview:1.6.1
+androidx.compose.ui:ui-unit-android:1.6.1
+androidx.compose.ui:ui-unit:1.6.1
+androidx.compose.ui:ui-util-android:1.6.1
+androidx.compose.ui:ui-util:1.6.1
+androidx.compose.ui:ui:1.6.1
+androidx.compose:compose-bom:2024.02.00
 androidx.concurrent:concurrent-futures:1.1.0
 androidx.core:core-ktx:1.12.0
 androidx.core:core:1.12.0
 androidx.customview:customview-poolingcontainer:1.0.0
 androidx.customview:customview:1.0.0
-androidx.emoji2:emoji2:1.4.0
+androidx.emoji2:emoji2:1.3.0
```

Comment:
Which dependency caused this downgrade?

Reply:
androidx.compose.ui:ui-text-android:1.6.1 has a transitive dependency on androidx.emoji2.emoji2.  The upper bound of versions supported changed from 1.4.0 to 1.3.0. 

Dependency information obtained using `./gradlew :app:dependencies > deps.txt`